bundles/org.openhab.binding.hive/src/main/java/org/openhab/binding/hive/internal/client/dto/SetpointDto.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.hive.internal.client.dto;
+
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ *
+ *
+ * @author Ross Brown - Initial contribution
+ */
+@NonNullByDefault
+public final class SetpointDto {
+    // TODO: Parse to DayOfWeek
+    public @Nullable Integer dayIndex;
+
+    // TODO: Parse to LocalTime
```

Comment:
can you do this todo?

Reply:
I left that for now because the schedule stuff is WIP.
I've started the low level API interaction bit for the client library but it's not used in the binding itself yet.
I'm exactly sure what I want to do with it (channels etc.).